lib/config/fileconf.go

```diff
@@ -1054,6 +1054,8 @@ type Database struct {
 	CACertFile string `yaml:"ca_cert_file,omitempty"`
 	// TLS keeps an optional TLS configuration options.
 	TLS DatabaseTLS `yaml:"tls"`
+	// Options are additional database options.
+	Options DatabaseOptions `yaml:"options"`
```

Comment:
Instead of generic "options", can we have a `mysql` section, e.g.

```yaml
mysql:
  server_version: x.y.z
```

Reply:
Changed